src/backend/sev/builder.rs

```diff
@@ -32,12 +33,16 @@ impl TryFrom<super::super::kvm::config::Config> for Builder {
         let kvm_fd = Kvm::new()?;
         let vm_fd = kvm_fd.create_vm()?;
 
-        let sev = Firmware::open()?;
+        let mut sev = Firmware::open()?;
+
+        // FIXME: use platform_status to get tcb_version and VCEK?
+        let _platform_status = sev.platform_status()?;
```

Comment:
If we discard the result value of this, why do we need to call it in the first place?
If there's something (SEV) implementation-specific going on, maybe worth adding a comment about it?
Otherwise, should we file an issue about the
> use platform_status to get tcb_version and VCEK

?

Reply:
Update the comment to:
```rust
        // FIXME: use `platform_status()` to prevent warnings about unused code
        // because we want to use  `platform_status()` later in a sub command.
```